src/editors/containers/TextEditor/components/ImageSettingsModal/hooks.js

```diff
@@ -7,6 +7,8 @@ import * as module from './hooks';
 export const state = {
   altText: (val) => React.useState(val),
   dimensions: (val) => React.useState(val),
+  isAltTextError: (val) => React.useState(val),
+  isAltTextValid: (val) => React.useState(val),
```

Comment:
I created two states for the error here.
isAltTextError corresponds with the alert at the top. It becomes true when save button is clicked. It becomes false when alert is dismissed.
isAltTextValid corresponds with the alt-text box and the error message near it. It becomes true or false only when the save button is clicked.

Reply:
This is... confusing.  Too close to each other without enough distinction. I would rename to something like:
 - showDismissibleError
 - showSubmissionError

Reply:
These sound better. I'll make a change for this.